pallet_p_options:
- initial: true
  name: Waiting for ready product to hold
  value: idle
- initial: false
  name: Robot's movement towards product
  value: Moving
- initial: false
  name: Gripper latches
  value: Latch
- initial: false
  name: Product is up
  value: Product up
- initial: false
  name: Robot's movement towards pallet
  value: Moving
- initial: false
  name: Gripper opens
  value: Gripper open
- initial: false
  name: Product is released
  value: Released
sub_options:
- initial: true
  name: Waiting for palletization process to start
  value: idle
- initial: false
  name: Waiting for product to arrive
  value: Arrival
- initial: false
  name: Product is ready to hold
  value: Product ready
- initial: false
  name: Product is up
  value: Product up
supervisor_options:
- initial: true
  name: IDLE
  value: idle
- initial: false
  name: Waiting for empty pallet to arrive
  value: Waiting
- initial: false
  name: Palletizing process
  value: Palletizing
- initial: false
  name: Waiting for full pallet to exit
  value: Exit
